.github/workflows/link-rot-checker.yml

```diff
@@ -0,0 +1,25 @@
+name: Link-rot-Checker
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+  - cron: "00 18 * * *"
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
Please use a hash instead of `v4`, see other workflows https://github.com/lima-vm/lima/blob/1abadcd2432f360936a6bc5de8e3cb01120ea805/.github/workflows/test.yml#L29